src/OrchardCore.Modules/OrchardCore.Users/Startup.cs

```diff
@@ -146,8 +139,51 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddScoped<IDisplayManager<User>, DisplayManager<User>>();
             services.AddScoped<IDisplayDriver<User>, UserDisplayDriver>();
             services.AddScoped<IDisplayDriver<User>, UserButtonsDisplayDriver>();
-            
+
             services.AddScoped<IDisplayDriver<ISite>, RegistrationSettingsDisplayDriver>();
         }
     }
+    
+    [Feature("OrchardCore.Users.Password")]
+    public class PasswordStartup : StartupBase
+    {
+        private const string ForgotPasswordPath = "ForgotPassword";
+        private const string ForgotPasswordConfirmationPath = "ForgotPasswordConfirmation";
```

Comment:
`ForgotPasswordConfirmationPath` and `ResetPasswordConfirmationPath` have the same value, this can't work.